core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -568,7 +568,7 @@ private[spark] class ExecutorAllocationManager(
       // We don't want to change our target number of executors, because we already did that
       // when the task backlog decreased.
       if (decommissionEnabled) {
-        val executorIdsWithoutHostLoss = executorIdsToBeRemoved.toSeq.map(
```

Comment:
No need to `toSeq` here because `executorIdsToBeRemoved(ArrayBuffer).map` finally calls toArray